Source/WebKit/Resources/SandboxProfiles/ios/com.apple.WebKit.WebContent.sb.in

```diff
@@ -1392,7 +1392,7 @@
     
 (allow mach-kernel-endpoint
     (apply-message-filter
-        (deny mach-message-send (with telemetry))
+        (deny mach-message-send)
```

Comment:
Likewise, should we just silence the one item we know is noisy but perhaps leave others?

Reply:
That is a very good point. I have adjusted the patch accordingly.

Thanks for reviewing!